lib/assetmanager.js

```diff
@@ -378,22 +396,15 @@
      * @param {Array.<string>} errors
      */
     AssetManager.prototype._reportSoftErrors = function (errors, component) {
-        
-        var namedComponent,
-            typeName;
-        
-        if (errors && errors.length > 0) {
-            if (!component.layer) {
-                typeName = "Layer Comp";
-            }
-            namedComponent = component.layer || component.comp;
-            
-            errors.forEach(function (err) {
-                this._errorManager.addError(namedComponent, err, typeName);
-            }.bind(this));
-
-            this._errorManager.reportErrors();
+        if (!errors || !errors.length) {
+            return;
         }
+
+        errors.forEach(function (err) {
```

Comment:
nice cleanup! If we had _.partial you could make this on line! (maybe you can with .bind() too but it would be a long line)